google/cloud/pubsub/internal/batch_callback.h

```diff
@@ -46,6 +46,9 @@ class BatchCallback {
 
   virtual void EndMessage(std::string const& ack_id,
                           std::string const& event) = 0;
+
+  virtual void AddEvent(std::string const& ack_id,
+                        std::string const& event) = 0;
```

Comment:
Should we list `AddEvent()` before `EndMessage()`?

Reply:
Ack, it does make more sense logically.